### DISCUSSION TITLE
Modernizing the app

### DISCUSSION THREAD

**Discussion by Nic787** [organicmaps/organicmaps#1725] [Ideas]
Hi,

I just discovered the app and it’s mostly based on MapsMe since it’ll the same devs. However, I feel like a bit of modernizing could be done:

1. New color palette of the map (going to white and some sort of green instead of the yellowish/grey palette).
2. Show speed limit when travelling based on OSM data.
3. New modern iconography.
4. Booking and rating integration like MapsMe.

Thank you

(I can show you example of new color if you want)

**Comment by biodranik** [comment]
1 and 3: you're welcome! Please show some examples, changing map styles and icons, and still keeping them readable in both light and dark styles is tricky.
2: check existing issues, it should be there, planned to fix.
4: why do you need booking in OM? How do you plan to maintain a high-quality user rating database for the free app?

**Reply by Nic787** [reply]
1 & 3:

I did something very fast and unfinished, but my idea is just to use lighter colors. A bit like Apple Map is doing with light/dark color, but with OM style and not a copy.
![image](https://user-images.githubusercontent.com/11185348/147039465-8b19d689-b7b1-463e-82cd-782404b4a32b.png) could make Highway a light orange/yellow instead of dark grey in my example. It would need some color testing to find out what work best for light/dark theme.

I think Apple Map nailed it in term of color. It’s easy on the eye.
![EB073476-0782-4FDD-9BCC-23E6EE8AFD77](https://user-images.githubusercontent.com/11185348/147121563-c50a6120-42db-4dfd-85f7-d4e25fb5e6ce.png)


Also, I'm not too sure about the +/- buttons. Do people use them?

2: Thanks! I didn't see that when testing. Also, is that possible to get the voice over read street name in the future instead of just "turn left and turn right"?

4: I've never used booking in MapsMe, but I think it could be really useful for someone travelling and looking quickly for a hotel nearby with a direct link to the booking.com page. I'm not really sure how it works, so can't say much... Here is a screenshot for example:
![image](https://user-images.githubusercontent.com/11185348/147035762-c337d0d7-6c95-4403-b28d-f7e7aadb6e2a.png)
Here is the rating system in MapsMe, but I have no idea how and if it works. I know you can also add a review. I guess they have money to  allow this.
![image](https://user-images.githubusercontent.com/11185348/147035896-50b1f9f8-c357-41cb-86e3-e7a98f4bb642.png)

**Reply by biodranik** [reply]
We don't need to "modernize" the style. It's very hard work to make it readable and consistent for many different object combinations.
What we need is different light/dark styles for different cases:
1. City tourism.
2. Hiking.
3. Cycling.
4. Car navigation.

See more issues about it here: https://github.com/organicmaps/organicmaps/issues?q=is%3Aissue+is%3Aopen+style+label%3AStyles

We plan to add street names. Please check existing issues.

Regarding booking, mapsme had a private deal with Booking to store its data offline. We can't do that legally. And we don't want to pollute the map.
